py4DSTEM/process/diffraction/crystal.py

```diff
@@ -660,7 +649,8 @@ def generate_diffraction_pattern(
             self.setup_diffraction(300e3)
 
         # Tolerance for angular tests
-        tol = 1e-6
+        # TODO should this be used somewhere ?
```

Comment:
This does seem to be unused, we can delete it

Reply:
removed